Resources/Prototypes/Entities/Objects/Fun/Tabletop/nanopoliafigurines.yml

```diff
@@ -0,0 +1,913 @@
+- type: entity
+  id: NanapoliaTabletopPiece
+  parent: BaseTabletopPiece
+  abstract: true
+  components:
```

Comment:
add `description: A figurine from nanopolia` to this and remove it from all the pieces

Reply:
Done